terraform/modules/kubernetes/kubernetes.tf

```diff
@@ -208,13 +218,13 @@ resource "kubernetes_config_map" "aggregate_job_config_map" {
     # BATCH_SIGNING_PRIVATE_KEY is a Kubernetes secret
     AWS_ACCOUNT_ID                       = data.aws_caller_identity.current.account_id
     BATCH_SIGNING_PRIVATE_KEY_IDENTIFIER = kubernetes_secret.batch_signing_key.metadata[0].name
+    INGESTOR_INPUT                       = "s3://${var.ingestion_bucket}"
     INGESTOR_IDENTITY                    = var.ingestion_bucket_role
-    INGESTOR_INPUT                       = var.ingestion_bucket
     INGESTOR_MANIFEST_BASE_URL           = var.ingestor_manifest_base_url
     INSTANCE_NAME                        = var.data_share_processor_name
-    OWN_INPUT                            = var.own_validation_bucket
+    OWN_INPUT                            = "gs://${var.own_validation_bucket}"
```

Comment:
Oof, I should have spotted the need for the scheme. I hope it wasn't a pain to debug this.